src/components/pages/events/eventPage/validEvent/ValidEvent.tsx

```diff
@@ -23,50 +25,102 @@ const ValidEventLayout: React.FC<{ event: Event }> = ({ event }) => {
   const history = useHistory();
 
   return (
-    <div className={styles.eventBody}>
+    <Flex
+      flexDir="column"
+      w="100%"
+      borderRadius="20px"
```

Comment:
Would prefer using `md`, `lg` or `xl` keys here (for example) instead of inline values for more universal styling